docs/value-props.md

```diff
@@ -0,0 +1,16 @@
+# Why Use Testkube?
```

Comment:
I like the initiative to add Problem & Solution to docs.
It's not quite there yet. Lets refine this better and delete this file from the PR for now.

Reply:
Done.